client/src/pages/mapper/MapLayers.js

```diff
@@ -7,10 +7,18 @@ import { treeHealth } from '../../util/treeHealth';
 const circleRadius = {
   'circle-radius': [
     'interpolate', ['linear'], ['zoom'],
-    12, 1,
-    18, 6,
+    12, 3,
```

Comment:
These radii make quite chunky circles, with a lot of overlap between trees.

Reply:
I think keeping them small when zoomed out is fine, as there's no way to pick one at that scale with something as pudgy as a finger.  The zoom range used for the interpolation could be increased, so they're bigger when zoomed in more than 18.  I don't think it helps tappability if they're overlapping.  It may be possible to increase the tap target without increasing the visible pixel width.

There's also something weird going on next to the adopt and plant buttons.  For about 2/3 the width of the screen, dragging there scrolls the whole page rather than moving the map.  I think there's something transparent sitting over the map.